prometheusalertmanager/prometheusalertmanager.go

```diff
@@ -0,0 +1,102 @@
+package prometheus
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/target/goalert/alert"
+	"github.com/target/goalert/integrationkey"
+	"github.com/target/goalert/permission"
+	"github.com/target/goalert/retry"
+	"github.com/target/goalert/util/errutil"
+	"github.com/target/goalert/util/log"
+	"github.com/target/goalert/validation/validate"
+)
+
+type alertmanagerPost struct {
```

Comment:
We can remove all the `json:"` tags here since Go will decode them regardless of case.